.github/workflows/CD.yml

```diff
@@ -0,0 +1,43 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - master
+      - feature/*
+  # Allows manual trigger of pipeline
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to AKS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2 
+
+      - name: Azure login
+        id: login
+        uses: azure/login@v1.4.3
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Set AKS context
+        id: set-context
+        uses: azure/aks-set-context@v3
+        with:
+          resource-group: 'cdp-pet-project' 
+          cluster-name: 'aks'
+
+      - name: Setup kubectl
+        id: install-kubectl
+        uses: azure/setup-kubectl@v3
+
+      - name: Kubectl apply
+        shell: bash
+        run: |
+          kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
```

Comment:
If memory serve me, we already discussed it but still just keep in mind it will be more fruitful to avoid code checkout in your case and publish bookingnfo.yaml as an artifact